x-pack/plugins/maps/public/connected_components/mb_map/mb_map.tsx

```diff
@@ -237,9 +240,14 @@ export class MBMap extends Component<Props, State> {
   }
 
   async _initializeMap() {
+    const initialView = await getInitialView(this.props.goto, this.props.settings);
```

Comment:
Did an audit in MBMap to find any other places were async actions were occurring without checks that component mounted.